<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11615" started-at="2020-11-22T18:39:24Z" finished-at="2020-11-22T18:39:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11615" started-at="2020-11-22T18:39:24Z" finished-at="2020-11-22T18:39:36Z">
      <class name="com.practise.AssertionTestFacebook">
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.practise.AssertionTestFacebook@1d057a39]" name="setUpTest" is-config="true" duration-ms="8225" started-at="2020-11-22T18:39:24Z" finished-at="2020-11-22T18:39:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="loginFb()[pri:3, instance:com.practise.AssertionTestFacebook@1d057a39]" name="loginFb" duration-ms="335" started-at="2020-11-22T18:39:33Z" finished-at="2020-11-22T18:39:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginFb -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.practise.AssertionTestFacebook@1d057a39]" name="tearDownTest" is-config="true" duration-ms="68" started-at="2020-11-22T18:39:33Z" finished-at="2020-11-22T18:39:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.practise.AssertionTestFacebook@1d057a39]" name="setUpTest" is-config="true" duration-ms="2792" started-at="2020-11-22T18:39:33Z" finished-at="2020-11-22T18:39:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="FAIL" signature="Homelogintitle()[pri:4, instance:com.practise.AssertionTestFacebook@1d057a39]" name="Homelogintitle" duration-ms="13" started-at="2020-11-22T18:39:36Z" depends-on-methods="com.practise.AssertionTestFacebook.loginFb" finished-at="2020-11-22T18:39:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Facebook – log in or sign up] but found [fghdfh]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Facebook – log in or sign up] but found [fghdfh]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.practise.AssertionTestFacebook.Homelogintitle(AssertionTestFacebook.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Homelogintitle -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.practise.AssertionTestFacebook@1d057a39]" name="tearDownTest" is-config="true" duration-ms="167" started-at="2020-11-22T18:39:36Z" finished-at="2020-11-22T18:39:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- com.practise.AssertionTestFacebook -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
